src/doc/unstable-book/src/compiler-flags/check-cfg.md

```diff
@@ -81,6 +81,7 @@ As of `2024-01-09T`, the list of known names is as follows:
 
 <!--- See CheckCfg::fill_well_known in compiler/rustc_session/src/config.rs -->
 
+ - `clippy`
```

Comment:
Would be great if you could also update the date above to `2024-02-15T`

Reply:
:+1: